simulation/simtypes/action.go

```diff
@@ -22,7 +22,7 @@ type Action interface {
 	// but this should not be the default.
 	Frequency() Frequency
 	Execute(*SimCtx, sdk.Context) (
-		OperationMsg simulation.OperationMsg, futureOps []simulation.FutureOperation, err error)
```

Comment:
Lets add a comment specifying that result data is data to be merkelized / we can compare consistency of across runs